### DISCUSSION TITLE
How to configure reporting on manufacturer specific attributes with ZBBridge

### DISCUSSION THREAD

**Discussion by CopyCat73** [arendst/Tasmota#14680] [Q&A] [answered]
I'm trying to figure out how to read the air quality value from an Ikea Starkvind. I understand that the bridge auto configures the reporting of standard attributes. The air quality however is in cluster 0xfc7d, attribute 0x0004. I should be sending something like ZbSend {"Device":"0xC345","Send":"FC7D_06/...."} 06 being the configure command, but I can't figure out the rest.. any help is appreciated!

**Comment by s-hadinger** [comment]
Do you have more details about the command or attribute? It's not a standard cluster number.

**Reply by CopyCat73** [reply]
These are the attributes according to the deconz people: https://github.com/dresden-elektronik/deconz-rest-plugin/issues/5351#issuecomment-968267581

I'm looking specifically at  0x0004 | u16 | r | Air Quality (PM2.5 in µg/m³).

**Comment by s-hadinger** [comment, marked as answer]
Thanks. It should be read attributes. I will get back to you shortly

**Reply by s-hadinger** [reply]
To read the attribute, use the following:

`ZbSend {"Device":"0xC345","Endpoint":1,"Cluster":"0xfc7d",""Read":4}

You don't need to specify the type while reading, the device returns the value with a type.

For attribute reporting it's a little more work. Unfortunately it's not in the doc (yet) but described here: https://github.com/arendst/Tasmota/pull/9050

**Comment by CopyCat73** [comment]
Thanks!
